src/elexmodel/models/BaseElectionModel.py

```diff
@@ -10,7 +10,6 @@
 warnings.filterwarnings("error", category=UserWarning, module="cvxpy")
 
 PredictionIntervals = namedtuple("PredictionIntervals", ["lower", "upper", "conformalization"], defaults=(None,) * 3)
-
```

Comment:
do you mind adding this white line back in, it's just that the `PredictionIntervals` and the logger don't go together.

Reply:
done